### PR TITLE
Add eslint:recommended to a usage example

### LINKED ISSUE #588: Mention eslint:recommended in eslint-plugin's README?
We were following [the README](https://github.com/typescript-eslint/typescript-eslint/blob/b5e95edbb811ba239d2856e9caee21e802fbd8fb/packages/eslint-plugin/README.md#usage) a bit too blindly and just copied this to our `.eslintrc.js`:

```js
extends: [
  'plugin:@typescript-eslint/recommended'
]
```

Only later we realized that this doesn't include any of the `eslint:recommended` rules :) Rookie mistake but we previously used TSLint which, if I remember correctly, set up everything.

I now realize that in the ESLint world, typescript-eslint is just piece of the puzzle but it was not immediately obvious to us. I'm not sure if we're an exception or not but maybe this could be mentioned in the README?

### PR DESCRIPTION
**Pull request by borekb**
Resolves #588.

### CODE REVIEW

File: packages/eslint-plugin/README.md [hunk 1]
```diff
@@ -47,7 +47,10 @@ You can also enable all the recommended rules at once. Add `plugin:@typescript-e
 
 ```json
```
Comment (bradzacher) [inline packages/eslint-plugin/README.md]: you can use `cjson` instead to add comments to the json which get highlighted properly

**Comment by ThomasdenH** [comment]
The recommended set is not entirely suitable for Typescript. Have a look at https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin/src/configs, there is a configuration that fixes that. It'd be great if you could add `"plugin:@typescript-eslint/eslint-recommended"`, since it clearly isn't documented well enough.

File: packages/eslint-plugin/README.md (hunk 1 above)
Comment (borekb) [reply packages/eslint-plugin/README.md]: I struggle to add it to the example in a way that I would entirely like. How about noting eslint:recommended separately, in a paragraph of text? Or if you have an edit in mind that you'd like, feel free to add it.

**Comment by borekb** [comment]
@ThomasdenH Good points, thanks! Will look into it later today.

File: packages/eslint-plugin/README.md (hunk 1 above)
Comment (borekb) [reply packages/eslint-plugin/README.md]: Updated in 86ae914a77adc48af7065ff85f426816fc90cc12.

**Comment by borekb** [comment]
@ThomasdenH I tried to experiment with how various configs behave on our codebase and it lead me to create #592. Will happily document integration with `eslint:recommended` once it's clear to me how it's best done.

**Comment by bradzacher** [comment]
The best integration would be:
```
[
  'eslint:recommended',
  'plugin:@typescript-eslint/eslint-recommended',
  'plugin:@typescript-eslint/recommended'
]
```

**Comment by borekb** [comment]
Great! Would you put it to the basic Usage section or dedicate a new one to it?

Another option is to link to [`eslint-plugin/src/configs`](https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin/src/configs) which already has a good README about the various configs and probably a better chance to stay up to date :)

**Comment by bradzacher** [comment]
I would say this should probably do a few things.
I doubt we'll add any more "recommended" style configs, and the configs readme is more there so I can direct people to it when they get their knickers in a knot over the recommended ruleset :)

Could you please do the following.

- change the wording from
```diff
- You can also enable all the recommended rules at once. Add `plugin:@typescript-eslint/recommended` in extends:
+ You can also enable all the recommended rules for our plugin. Add `plugin:@typescript-eslint/recommended` in extends:
```

- add a new "section" below that along the lines of
```
You can also use eslint recommended with this plugin. Add both `eslint:recommended ` and `plugin:@typescript-eslint/eslint-recommended`:
[
  'eslint:recommended',
  'plugin:@typescript-eslint/eslint-recommended',
  'plugin:@typescript-eslint/recommended'
]
```

**Comment by borekb** [comment]
@bradzacher great suggestions, thank you! Updated in 2e6f60e.

**Pull request merged**